solutions/usaco-737.mdx

```diff
@@ -0,0 +1,53 @@
+---
+id: usaco-737
+source: USACO Bronze 2017 US Open
+title: Modern Art
+author: Juheon Rhee
+---
+
+[Official Analysis (C++)](http://www.usaco.org/current/data/sol_art_bronze_open17.html)
+
+## Implementation
+**Time Complexity**: $\mathcal O(N^3)$
+
+<LanguageSection>
+
+<PySection>
+
+```py
+res = [[0] * 11 for _ in range(11)]
```

Comment:
magic number
put 10 in a variable like WIDTH or something (all caps)

Reply:
okay.